tox.ini

```diff
@@ -0,0 +1,35 @@
+[tox]
+requires =
+    tox-conda
+
+[testenv]
+conda_spec =
+    py38: {toxinidir}{/}requirements{/}locks{/}py38-linux-64.lock
+    py39: {toxinidir}{/}requirements{/}locks{/}py39-linux-64.lock
+    py310: {toxinidir}{/}requirements{/}locks{/}py310-linux-64.lock
+usedevelop =
+    true
+
+[testenv:py{38,39,310}-tests]
+description = Perform Iris unit, integration and system tests.
+commands =
+    pytest -n auto {posargs} {toxinidir}{/}lib{/}iris{/}tests
+
+[testenv:py{38,39,310}-docs{,-linkcheck,-tests}]
+description = Build, and optionally linkcheck or test, the Iris documentation
+allowlist_externals =
+    make
+changedir =
+    {toxinidir}{/}docs
+commands =
+    make clean html
+    linkcheck: make linkcheck
+    tests: make doctest
```

Comment:
This results in the following py3.10 jobs:
py310-docs: simply build the documentation
py310-linkcheck: build the docs and run the link check
py310-tests: build the docs and run the tests